src/System.Net.Requests/tests/System.Net.Requests.Tests.csproj

```diff
@@ -10,7 +10,8 @@
     <ProjectGuid>{851A40FE-7F07-415D-8592-5FE2096E84D3}</ProjectGuid>
```

Comment:
@weshaggard, thanks for the great explanation!  Appreciated.  Property group deleted.